var/spack/repos/builtin/packages/fujitsu-fftw/package.py

```diff
@@ -33,10 +33,12 @@ class FujitsuFftw(FftwBase):
     homepage = "https://github.com/fujitsu/fftw3"
     url = "https://github.com/fujitsu/fftw3/archive/sve-v1.0.0.tar.gz"
 
+    version('1.1.0', sha256='47b01a20846802041a9533a115f816b973cc9b15b3e827a2f0caffaae34a6c9d')
     version('1.0.0', sha256='b5931e352355d8d1ffeb215922f4b96de11b8585c423fceeaffbf3d5436f6f2f')
 
     variant('shared', default=True, description='Builds a shared version of the library')
     variant('openmp', default=True, description="Enable OpenMP support")
+    variant('threads', default=True, description="Enable thread support")
```

Comment:
How do `openmp` and `threads` play with each other? Is it possible to have `~threads+openmp`?

Reply:
@otsukay

Reply:
No worries, thanks for the reply!